src/ape/options.py

```diff
@@ -31,3 +36,31 @@ def verbose_option(help=""):
         default=False,
         help=help,
     )
+
+
+class Alias(click.Choice):
+    """Wraps ``click.Choice`` to load account aliases for the active project at runtime."""
+
+    name = "alias"
+
+    def __init__(self, account_type: Type[AccountAPI] = KeyfileAccount):
+        # NOTE: we purposely skip the constructor of `Choice`
+        self.case_sensitive = False
+        self._account_type = account_type
+
+    @property
+    def choices(self) -> List[str]:  # type: ignore
+        # NOTE: This is a hack to lazy-load the aliases so CLI invocation works properly
+        return [a.alias for a in accounts.get_accounts_by_type(self._account_type)]
+
+
+def _require_non_existing_alias(arg):
```

Comment:
This callback, however, still checks against all the aliases...

I am not sure how important that it, it _could_ probably still work to have aliases with the same name in separate containers. That seems out of scope though, and not that important...

Reply:
no, the aliases are a global construct, so we want to make sure they don't collide